### PR TITLE
Add window and mobile version popup 

### PR DESCRIPTION
**Pull request by abreham1222**
1. Added window popup for a desktop version
2. Added mobile version popup
3. Added linters for js
4. Added other images

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -9,4 +9,101 @@ humburger.addEventListener('click', () => {
...
+  titleTwo: 'Multi Post Stories',
+  title: 'Keeping track of hundreds  of components website',
+  description_first: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+  image_slideshow: '/images/snapshoot.png',
+  languages: {
+    language_1: 'html',
+    language_2: 'Bootstrap',
+    language_3: 'Ruby on Rails',
+  },
+};
```
Comment (vic778) [inline script.js:82]: kindly all your projects should be implemented in your js file. you have to create an array that will contain all of your projects like projects = 
`[
   {
   },
  {
  },
  ....
] `

File: script.js [hunk 2]
```diff
@@ -9,4 +9,101 @@ humburger.addEventListener('click', () => {
...
+  parent.style.display = 'block';
+  sect.style.filter = 'blur(10px)';
+}
+
+btn.addEventListener('click', appear);
+
+function disappearX() {
+  parent.style.display = 'none';
+  section.style.filter = 'blur(0px)';
+}
```
Comment (vic778) [inline script.js:59]: please make sure you're following the design of the project. [click here](https://github.com/abreham1222/Microverse-portfolio.git)to see the design of your Figma template. your see button should pop up the image with the description according the requirement of the project

**Review by vic778 — CHANGES_REQUESTED** [review]
## Hi @abreham1222 ,

Good job so far!

## Highlights!
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Good code organization ✔️
- All tests passed ✔️ 

Hello ✋, you have done well reaching this point of the project 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@victoir` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Davidosky007 — COMMENTED** [review]
Hello @abreham1222 👋,

_This is an additional review as a result of the second opinion request._

Well done with all the hard work you guys have put into this milestone 👏 💪 

- Kindly note that this  `[ ]` is what represents an array and `{ }` is what wraps an object, according to the project requirement you should have an array of objects `[ { }, { }, .... ]` kindly add more project objects so that you can loop through it and render each on the page dynamically 👍 
- Also on the desktop the see project button doesn't  display a popup on click and on MObile some see project buttons also are not working.

Kindly 🙏 Fix these issues and submit another Code review request.
Please ask for another review in your Dashboard and the next Code Reviewer will review it accordingly 

Keep rocking 🚀!

File: popup.css [hunk 3]
```diff
@@ -51,14 +52,14 @@
...
-    padding-right: 30px;
-    padding-left: 10px;
   }
 
   .modal-list li {
     border: 1px solid rgb(194, 188, 188);
+    padding-right: 10px;
+    padding-left: 10px;
+    padding-top: 5px;
+    padding-bottom: 5px;
```
Comment (brytebee) [inline popup.css:62]: - [ ] **Required:** You have done an amazing job so far. 👏 
However, there are some changes you need to make.

Your Design | Template
:---: | :---:
image | image

1. Your image overflows from the modal, kindly fix this.

2. Your modal has no contrast when popped up from the main page. It is better for UX and accessibility to have a contrast. Kindly make the main page dim when the modal is popped as illustrated in the template.

3. **Optional** You could consider reducing the height of the mobile modal and truncating the text to have a better UI design.

4. On the mobile pop-up, re-position the buttons to be more like the template, justify the text and dim the main page.

Your Design | Template
:---: | :---:
image | image

File: script.js [hunk 4]
```diff
@@ -9,4 +9,154 @@ humburger.addEventListener('click', () => {
...
+    const paragraph = document.querySelector('.modal-lorem');
+    paragraph.textContent = projectDetail.descriptionOne;
+    const picture = document.querySelector('.img-shoot');
+    picture.src = projectDetail.imageOne;
+    const languag = document.querySelectorAll('.modal-listed');
+    languag.forEach((li, s) => {
+      li.textContent = projectDetail.languages[s];
+    });
+    const live = windowPop.querySelector('.see-live')[0];
+    live.href = projectDetail.linklive;
```
Comment (brytebee) [inline script.js:152]: - [ ] **Required:** Kindly fix these console errors.

image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @ 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- All linter checks are installed and pass.✔️
- The popup button opens the modal and the **X** closes it. ✔️
- Design is responsive to screen size (mobile & desktop). ✔️
- Handled JS Array of Object correct and render data to the DOM dynamically. ✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**